### PR TITLE
PLAT-126940: Enable wheeling for Slider 

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
From 2022 TV UX/GUI workshop, UX wants to enable wheeling on Slider.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fix Slider to enable wheeling 

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
To avoid console error or console warning, I used 'passive:false'option.
```
$0.addEventListener("wheel", (ev)=>ev.preventDefault()); // $0 === any Node
```
```
[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/6662647093133312
```
```
VM2717:1 [Violation] Added non-passive event listener to a scroll-blocking 'wheel' event. 
Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952
```

### Links
[//]: # (Related issues, references)
PLAT-126940

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
 - `sandstone/Item` public class names `itemContent`, `content`, and `label`
 - `sandstone/Scroller` prop `scrollbarTrackCss` to customize scroll track and thumb style
 
+### Changed
+
+- `sandstone/Slider` to enable wheeling to change the value
```
Comment (0x64) [inline CHANGELOG.md:15]:
```suggestion
- `sandstone/Slider` to change the value by wheeling
```

**Review by 0x64 — APPROVED** [review]
:lgtm:

**Review by MikyungKim — COMMENTED** [review]
Please rebase your branch with the latest develop.

File: Slider/utils.js [hunk 2]
```diff
@@ -88,6 +96,25 @@ const handleDecrement = handle(
 	emitChange(-1)
 );
 
+
```
Comment (MikyungKim) [inline Slider/utils.js:99]: Let's remove the one line.

**Review by MikyungKim — APPROVED** [review]
:shipit:

**Pull request merged**